agent/agentscripts/agentscripts.go

```diff
@@ -172,22 +179,31 @@ func (r *Runner) StartCron() {
 	}
 }
 
+type ExecuteOption int
+
+const (
```

Comment:
```suggestion
// ExecuteOption enums.
const (
```

In line with earlier nit, this is sufficient for this section but feel free to expand with more details if you want.